safehttp/form_test.go

```diff
@@ -12,1135 +12,476 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package safehttp_test
+package safehttp
 
 import (
 	"math"
-	"net/http"
-	"net/http/httptest"
 	"strconv"
-	"strings"
 	"testing"
-	"text/template"
 
 	"github.com/google/go-cmp/cmp"
-	"github.com/google/go-safeweb/safehttp"
 )
 
-func TestFormValidInt(t *testing.T) {
-	stringMaxInt := strconv.FormatInt(math.MaxInt64, 10)
+func TestFormValidInt64(t *testing.T) {
 	tests := []struct {
-		name string
-		req  *http.Request
+		val  string
 		want int64
 	}{
-		{
-			name: "Valid int in POST non-multipart request",
-			req: func() *http.Request {
-				postReq := httptest.NewRequest("POST", "http://foo.com/", strings.NewReader("pizza="+stringMaxInt))
-				postReq.Header.Set("Content-Type", "application/x-www-form-urlencoded")
-				return postReq
-			}(),
-			want: math.MaxInt64,
-		},
-		{
-			name: "Valid int in POST multipart request",
-			req: func() *http.Request {
-				multipartReqBody := "--123\r\n" +
-					"Content-Disposition: form-data; name=\"pizza\"\r\n" +
-					"\r\n" +
-					stringMaxInt + "\r\n" +
-					"--123--\r\n"
-				multipartReq := httptest.NewRequest("POST", "http://foo.com/", strings.NewReader(multipartReqBody))
-				multipartReq.Header.Set("Content-Type", `multipart/form-data; boundary="123"`)
-				return multipartReq
-			}(),
-			want: math.MaxInt64,
-		},
+		{val: "123", want: 123},
+		{val: "9223372036854775807", want: math.MaxInt64},
```

Comment:
This is optional, but I'd use `strconv...` to make the input, same for other tests.

Reply:
I don't agree, that's harder to read.